harness/determined/experimental/client.py

```diff
@@ -295,7 +296,15 @@ def logout() -> None:
 
 @_require_singleton
 def list_users(active: Optional[bool] = None) -> List[User]:
-    """Get a list of all Users."""
+    """Get a list of all Users.
+
+    Arg:
+        active: if this parameter is set to True, filter for active users only.
+            When false, filter for inactive users. Return all users otherwise.
+
+    Returns:
+        :class:`~determined.common.experimental.user.User`.
```

Comment:
nit: actually returns a list of them

Reply:
Also, could you make please prefer `determined.experimental.client.User` to `determined.common.experimental.user.User` path? The former is the public path, even if Sphinx autodoc doesn't always pick the right one in the existing docs.

Tested on my system, link seems fine after changing to `client.User`.